src/pmi/PMIDriver.cs

```diff
@@ -167,7 +167,7 @@ private static PrepAllResult Compute(PrepAllInfo pi)
                 p.StartInfo.RedirectStandardError = true;
 
                 // Fetch our command line. Split off the arguments.
-#if NETCOREAPP2_1
+#if NETCOREAPP2_1 || NETCOREAPP2_2 || NETCOREAPP3_0
                 // For .Net Core the PMI assembly is an argument to dotnet.
                 string newCommandLine = Environment.CommandLine.Replace("DRIVEALL", "PREPALL");
 #else
```

Comment:
It's kind of surprising if this is the best way to distinguish full from core.  Would it be better to add the known full fx symbols on an #elif here so you can #error for next time?

Reply:
(Just saw the comment about NETSTANDARD2_1, which mitigates my comment above somewhat)

Reply:
I don't know if it's the best way. Maybe we should ask around.

It is more annoying to have to keep stringing together those netcoreapps.